### ISSUE TITLE
Almost identical strings

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/gene-expression-essentials#131]
When doing the ES6 module feasibility tests, this warning came up:

> client:135 ./build/strings/GENE_EXPRESSION_ESSENTIALS_mRnaDestroyer.js
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
> * /Users/jon/phet/git/chipper/build/strings/GENE_EXPRESSION_ESSENTIALS_mRnaDestroyer.js
    Used by 2 module(s), i. e.
    /Users/jon/phet/git/gene-expression-essentials/js/multiple-cells/view/MultipleCellsScreenView.js
> * /Users/jon/phet/git/chipper/build/strings/GENE_EXPRESSION_ESSENTIALS_mrnaDestroyer.js
    Used by 2 module(s), i. e.
    /Users/jon/phet/git/gene-expression-essentials/js/manual-gene-expression/view/BiomoleculeToolboxNode.js

It looks like there are two almost-identical strings that should be collapsed into one.

**Comment by jbphet** [comment]
@jonathanolson - the nearly-identical strings have been consolidated.  Please take a look and, if this resolves the concern related to ES6 module feasibility, please close.

**Comment by jbphet** [comment]
I should probably note for the record that I modified several translations by hand, and some of those translations only included a value for the string that was being removed (mrnaDestroyer), and not the one that was being left in place (mRnaDestroyer).  I renamed the string key in those files, and then forced a rebuild so that all translations would be up to date.  The sims for which the renaming of the string key (and the subsequent forced redeploy) was necessary were es_ES, fa, iw, ja, mk, ta, and tk.
